site/sidebar.yml

```diff
@@ -0,0 +1,12 @@
+sidebar:
+  - title: unused 
```

Comment:
Is “unused” a special title for scaladoc? How does the sidebar look like?

Reply:
This is due to [this issue](https://github.com/lampepfl/dotty/issues/12969) and [this issue](https://github.com/lampepfl/dotty/issues/12983). Briefly, currently scaladoc does not pick up the title specified in sidebar.yml but only the one defined in frontmatter. However not specifying a title in the sidebar results in a crash.

If no title is specified in `sidebar.yml` or in the `title` frontmatter property, scaladoc uses the name of the file as sidebar entry